e2e/various-suite/loki-editor.spec.ts

```diff
@@ -8,7 +8,7 @@ const addDataSource = () => {
       'Unable to connect with Loki (Failed to call resource). Please check the server logs for more details.',
     name: dataSourceName,
     form: () => {
-      e2e.components.DataSource.DataSourceHttpSettings.urlInput().type('http://loki-url:3100');
+      e2e().get('#connection-url').type('http://loki-url:3100');
```

Comment:
Here I wish we had selectors, but that would have to be implemented in Connection Settings. Maybe it would be worth to create draft PR and ask the team who maintains the component for review.

Reply:
I agree. I couldn't replace the original one and didn't want to add something like `newUrlInput`.

> Maybe it would be worth to create draft PR and ask the team who maintains the component for review.

Sure, let's do that.